program/rust/src/deserialize.rs

```diff
@@ -112,3 +117,29 @@ pub fn initialize_pyth_account_checked<'a, T: PythAccount>(
 
     load_account_as_mut::<T>(account)
 }
+
+// Creates pda if needed and initializes it as one of the Pyth accounts
+pub fn create_pda_if_needed<'a, T: PythAccount>(
+    account: &AccountInfo<'a>,
+    funding_account: &AccountInfo<'a>,
+    system_program: &AccountInfo<'a>,
+    program_id: &Pubkey,
+    seeds: &[&[u8]],
+    version: u32,
+) -> Result<(), ProgramError> {
+    let target_rent = Rent::default().minimum_balance(T::MINIMUM_SIZE);
```

Comment:
you have to use `Rent::get()` not `Rent::default()` because we changed the minimum rent on Pythnet. (We just ran into this problem in the attester code)

Reply:
Can we do this in a future PR?